ansible/configs/sap-hana/default_vars.yml

```diff
@@ -89,32 +90,34 @@ ansible_tower:
-      groups:
-      - name: "sap"
-        hosts:
-        - name: "hana1"
-        - name: "hana2"
-        - name: "s4hana"
-      - name: "hanas"
-        hosts:
         - name: "hana1"
+          description: "Primary SAP HANA Host"
+          variables: '---\nstorage_pools:\n  - name: sap\n    disks:\n      - {{ sap_extra_device }}\n    volumes:\n      - name: data\n        size: \"128 GiB\"\n        mount_point: \"/hana/data\"\n        state: present\n      - name: log\n        size: \"64 GiB\"\n        mount_point: \"/hana/log\"\n        state: present\n      - name: shared\n        size: \"256 GiB\"\n        mount_point: \"/hana/shared\"\n        state: present\n      - name: sap\n        size: \"50 GiB\"\n        mount_point: \"/usr/sap\"\n        state: present\nsap_hana_deployment_bundle_path: /software/HANA_installation\nsap_hana_deployment_bundle_sar_file_name: IMDB_SERVER20_046_0-80002031.SAR\nsap_hana_deployment_sapcar_path: /software/SAPCAR\nsap_hana_deployment_sapcar_file_name: SAPCAR_1311-80000935.EXE\nsap_hana_deployment_root_password: \"{{ master_sap_password }}\"\nsap_hana_deployment_sapadm_password: \"{{ master_sap_password }}\"\nsap_hana_deployment_hana_sid: RHE\nsap_hana_deployment_hana_instance_number: \"00\"\nsap_hana_deployment_hana_env_type: development\nsap_hana_deployment_hana_mem_restrict: \"n\"\nsap_hana_deployment_common_master_password: \"{{ master_sap_password }}\"\nsap_hana_deployment_sidadm_password: \"{{ master_sap_password }}\"\nsap_hana_deployment_hana_db_system_password: \"{{ master_sap_password }}\"\nsap_hana_deployment_ase_user_password: \"{{ master_sap_password }}\"\nsap_hana_deployment_apply_license: false'
```

Comment:
This is definitely not readable... all these newlines should be expanded so that the settings can actually be read.